rx-preferences/src/main/java/com/f2prateek/rx/preferences2/RealPreference.java

```diff
@@ -80,7 +79,9 @@
   }
 
   @Override public void delete() {
```

Comment:
Don't mind, just read that after aapply the next calls to the preference will be blocking till the async operation returns.